main.go

```diff
@@ -163,7 +163,9 @@ func main() {
 		}()
 	}
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	restConfig := ctrl.GetConfigOrDie()
+	restConfig.UserAgent = remote.DefaultClusterAPIUserAgent("cluster-api-controller")
```

Comment:
Above with use `-manager` whereas here we use `-controller`, can we consolidate on using the same format everywhere?